### PR TITLE
chore: docs, cloud deployment

### PR DESCRIPTION
**Pull request by YuriBuerov**
#### Checklist

- [ ] run `make test`
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs-website/src/pages/docs/cloud/deployments.md [hunk 1]
```diff
@@ -1,38 +1,238 @@
...
+    "@wundergraph/sdk": "^0.133.1"
+  },
+  "devDependencies": {
+    "@types/node": "^14.14.37",
+    "typescript": "^4.8.2"
+  }
+}
+```
+
+As you can see, we have a `build` script, which is executed during the deployment process.
```
Comment (StarpTech) [inline docs-website/src/pages/docs/cloud/deployments.md:124]: `during build`

File: docs-website/src/pages/docs/cloud/deployments.md [hunk 2]
```diff
@@ -1,38 +1,238 @@
...
+  }
+}
+```
+
+If you don't have the `.wundergraph` directory, the build script would look like this:
+
+```json
+{
+  "scripts": {
+    "build": "wunderctl generate --debug --pretty-logging=true --wundergraph-dir ."
```
Comment (StarpTech) [inline docs-website/src/pages/docs/cloud/deployments.md:142]: Point to the `.` at h the end of the command.

File: docs-website/src/pages/docs/cloud/deployments.md [hunk 3]
```diff
@@ -1,38 +1,238 @@
...
+  }
+}
+```
+
+However, as mentioned above, if you don't have a build script,
+WunderGraph Cloud will automatically detect the path to the wundergraph dir, execute `wunderctl generate` command by default.
```
Comment (StarpTech) [inline docs-website/src/pages/docs/cloud/deployments.md:148]: Should we mention the exact command here?

**Review by StarpTech — CHANGES_REQUESTED** [review]
Awesome :+1:

File: docs-website/src/pages/docs/cloud/deployments.md [hunk 4]
```diff
@@ -1,38 +1,238 @@
...
+The WunderGraph Cloud officially supports `turborepo`, which is a type of monorepo that allows for faster and more efficient builds.
+If you have a `turbo.json` at the root of your repository, no additional steps are required for the deployment.
+
+To test this feature, you can [fork our turborepo example repository](https://github.com/wundergraph/cloud-starter-turbo/fork).
+
+Follow the steps from the `Import your repository` and `Deploy your project`.
+
+# Redeploy
+
+- Make changes to the `main` branch, e.g. change the query in `.wundergraph/operations/`.
```
Comment (fiam) [inline docs-website/src/pages/docs/cloud/deployments.md:88]: `change the query` `change a query`?

File: docs-website/src/pages/docs/cloud/deployments.md (hunk 2 above)
Comment (YuriBuerov) [reply docs-website/src/pages/docs/cloud/deployments.md:142]: what do you mean? in this case, it's correct: https://github.com/wundergraph/cloud-starter-turbo/blob/main/apps/api/package.json#L7

**Review by fiam — APPROVED** [review]
LGTM

File: docs-website/src/pages/docs/cloud/deployments.md (hunk 3 above)
Comment (StarpTech) [reply docs-website/src/pages/docs/cloud/deployments.md]: ok

File: docs-website/src/pages/docs/cloud/deployments.md (hunk 2 above)
Comment (StarpTech) [reply docs-website/src/pages/docs/cloud/deployments.md:142]: I mean, the dot can be easily missed. I'd add a comment.

**Pull request merged**